frontend/src/metabase-lib/Question.ts

```diff
@@ -313,43 +314,43 @@ class QuestionInner {
     return this._card && this._card.displayIsLocked;
   }
 
-  maybeResetDisplay(sensibleDisplays, previousSensibleDisplays): Question {
+  maybeResetDisplay(
+    data: DatasetData,
+    sensibleDisplays: [string],
+    previousSensibleDisplays: [string] | null,
```

Comment:
:x: This array can have more than 1 element, and will be `undefined` when default `[{}]` value is used here: https://github.com/metabase/metabase/blob/f9a3b132c4c5654f5cf722654b43a58dc00de92d/frontend/src/metabase/query_builder/actions/querying.js#L178

```suggestion
    previousSensibleDisplays: string[] | undefined
```

Will it ever be `null`?